projects/go-lib/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "@tangoe/goponents",
-  "version": "1.5.9",
+  "version": "1.6.0",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mobi/goponents.git"
   },
   "author": "Tangoe Mobile",
-  "description": "A library of UI components for Angular v7+.",
```

Comment:
I we no longer compatible with v7?

Reply:
We are, but this is more of a recommendation for use right? If we're targeting 8+ then I think we should be consistent here.